### PR TITLE
Fixing bug in mbs to pass general bandpass to mbs

### PR DESCRIPTION
**Pull request by sgiardie**
Related to issue #269

### CODE REVIEW

**Comment by ziotom78** [comment]
Sorry, I closed issue #269 not realizing that this PR was not merged…#281 

@sgiardie , I had just one minor comment about the code (see above), once this is answered you can surely merge it!

**Review by ziotom78 — COMMENTED** [review]
There is just one comment about the use of dictionaries via `.get` instead of `[]`, so that we are robust against missing keys. Other than that, everything looks ok to me!

File: litebird_sim/mbs/mbs.py
[the remark this answers is not in the document]
Comment (sgiardie) [reply litebird_sim/mbs/mbs.py:1]: Hi @ziotom78, sorry I saw your reply just now!
I changed it as you suggested. Can I merge it now?

[the remark this answers is not in the document]
Comment (ziotom78) [reply litebird_sim/mbs/mbs.py:1]: Yes, of course! Be sure to update the CHANGELOG before doing the merge. Thanks a lot!

**Pull request merged**